### PR TITLE
Add Photoswip support

### PR DESCRIPTION
**Pull request by WengerK**

### CODE REVIEW

File: assets/js/uww.js [hunk 1]
```diff
@@ -50,24 +50,28 @@
...
 				var $containerBlock = $('.node-type-article .media-gallery-media, .node-type-event .media-gallery-media').packery({
-					itemSelector: '.field-item',
+					itemSelector: 'figure',
 	  				gutter: 0,
 	  				isHorizontal: true,
 	  				containerStyle: null
 				});
-				$containerBlock.imagesLoaded( function() {
+
+        $containerBlock.imagesLoaded( function() {
```
Comment (gido) [inline assets/js/uww.js:70]: CS: indentation looks bad here

**Comment by gido** [comment]
Looks good 👍

File: assets/js/uww.js (hunk 1 above)
Comment (WengerK) [inline assets/js/uww.js:70]: It's because this file is a mess between spaces and tabs 💃

**Comment by WengerK** [comment]
I'm not very proud of #4702821 unfortunately we can't do better. Indeed, this page is provided by the module Media Gallery and it's impossible (seriously the markup isn't wrapped in anything) to move the pagination to an other place.

It's also impossible to change the number of item by row. Because nothing is in a wrapped markup but everything inline.

A better solution is to create a new page with a view.

**Pull request merged**